packages/next/src/server/load-components.ts

```diff
@@ -86,11 +86,11 @@ async function loadClientReferenceManifest(
   manifestPath: string,
   entryName: string
 ): Promise<ClientReferenceManifest | undefined> {
-  process.env.NEXT_MINIMAL
-    ? // @ts-ignore
-      __non_webpack_require__(manifestPath)
-    : require(manifestPath)
   try {
+    process.env.NEXT_MINIMAL
```

Comment:
What is this fixing?

Reply:
Since I'm no longer throwing a build error if there are two matching `children` parallel segments, appPaths will technically contain 2 pages and will throw an error when attempting to `require` it because the `FlightManifestPlugin` will have only generated a manifest for a single entrypoint. Since the other entry won't be used it seems safe to ignore, and `loadClientReferenceManifest` could potentially return undefined so I just lifted the try a bit higher.